File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.java
import java.util.*;

class Solution {
    public String solution(String s) {
        String answer = "";
        String[] strArr = s.split(" ");
        
        int n = strArr.length;
        int[] intArr = new int[n];
        
        for(int i=0; i<n; ++i){
            intArr[i] = Integer.parseInt(strArr[i]);
        }
        
        Arrays.sort(intArr);
        
        answer = intArr[0] + " " + intArr[n-1];
        return answer;
    }
}

File: 프로그래머스/2/12939. 최댓값과 최솟값/최댓값과 최솟값.py
def solution(s):
    answer = ''
    str_arr = s.split(" ")
    int_arr = [int(sa) for sa in str_arr]
    int_arr.sort()
    
    answer = str(int_arr[0]) + " " + str(int_arr[-1])
     
    return answer